tests/test_similarity_dice.py

```diff
@@ -0,0 +1,253 @@
+import unittest
```

Comment:
This file is mostly the same as test_similarity.py, but it uses the new API instead of the old API.